ui/shared/components/form/FormWizard.jsx

```diff
@@ -42,11 +48,17 @@ class FormWizard extends React.PureComponent {
       submitButtonText: 'Next',
       submitButtonIcon: 'angle double right',
     }
+    const backButtonProps = pageIndex === 0 ? {} : {
+      onCancel: this.navigateBack,
+      cancelButtonText: 'Back',
+      cancelButtonIcon: 'angle double left',
```

Comment:
The double left angle looks like returning to the beginning. Should a single angle left be better?

Reply:
No, I would like to keep this as is to keep it consistent with the "Next" button